go-controller/pkg/cluster/gateway_localnet.go

```diff
@@ -167,7 +167,7 @@ func initLocalnetGatewayInternal(nodeName string, clusterIPSubnet []string,
 	}
 
 	err = util.GatewayInit(clusterIPSubnet, nodeName, ifaceID, localnetGatewayIP,
-		macAddress, localnetGatewayNextHop, subnet, true, 0, nodePortEnable)
+		macAddress, localnetGatewayNextHop, subnet, true, config.Gateway.VLANID)
```

Comment:
Instead of config.Gateway.VLANID it should be 0 since VLAN ID doesn't apply for local gateway mode.